ckanext/who_afro/assets/css/who-afro.css

```diff
@@ -1739,3 +1739,179 @@ p.learn-more a {
 .promoted-right {
     z-index: 2;
 }
+
+
+/* dev banner */
+.dev-banner-container {
+    display: none;
+    width: 100%;
+}
+
+.nav-for-large-screen {
+    display: block;
+}
+
+.account-masthead .account ul li.dev-banner {
+    display: none;
+}
+
+.dev-banner-mobile {
+    display: none;
+}
+
+
+@media only screen and (max-width: 627px) {
+
+    .dev-banner-container .row {
+        --bs-gutter-x: 0;
+    }
+
+    .account-masthead {
+        height: 100%;
+    }
+
+    .dev-banner-container {
+        display: block;
+    }
+
+    .nav-for-large-screen {
+        display: none;
+    }
+
+    .account-masthead .container {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+        padding: 0;
+    }
+
+    .list-unstyled {
+        width: 100%;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    .list-unstyled li {
+        width: 100%;
+    }
```

Comment:
When working with CSS you have to be so careful when choosing your rules.  I can see you have added in some .list-unstyled formatting for the language selector.  But there are .list-unstyled objects everywhere in the site, which you have potentially then broken. When ever writing CSS you need to think very carefully about what the rule should be applied to in the site.  And when in any doubt at all, write a really specific rule that couldn't possibly apply to anything else e.g. .dev-banner-container .list-unstyled so that you know you are not unintentionally breaking something somewhere else in the site.  It's safer to be over-specific with your CSS rules, and then ease them as and when you see possible efficiencies, rather than to write vague rules that could break parts of the site without you being aware.